### ISSUE TITLE
Mark as archived or deprecated

### ISSUE THREAD

**Issue by skillsnate** [twitter/typeahead.js#1816]
When looking for a valid ajax supported select solution, this one looks good, until you see the demo is broken and the travis builds are failing, and the last closed issue was in 2021. Can we have someone mark this as Archived or Deprecated please?

**Comment by gumbarros** [comment]
@elonmusk X sucks

**Comment by petermorlion** [comment]
Agreed. In the meantime, what alternatives have you been using? I'm looking to update an old application to Bootstrap 5 and still need an autocomplete/typeahead input.

**Comment by gumbarros** [comment]
> Agreed. In the meantime, what alternatives have you been using? I'm looking to update an old application to Bootstrap 5 and still need an autocomplete/typeahead input.

@petermorlion If you find another alternative please let me know. I'm also needing autocomplete at BS5. I'm still using typeahead but this fork: https://github.com/corejavascript/typeahead.js

**Comment by petermorlion** [comment]
For now, I've added the typeahead javascript file from [here](https://twitter.github.io/typeahead.js/) but already had to modify it a bit.

**Comment by petermorlion** [comment]
Update: I've switched to this library with success: https://github.com/lekoala/bootstrap5-autocomplete
